samples/database.sample.yml

```diff
@@ -0,0 +1,670 @@
+      definition: s/he lights (it/him) for s.o.
+  specificWordClass: VTA
+
+- fstStem:          mâcisw
+  generalWordClass: V
+  key:              mâciswêw
+  lemma:
+    sro: mâciswêw
+    cw:  mâciswêw
+  senses:
+    - category:   VTA-3
+      definition: s/he cuts s.o. open
+  specificWordClass: VTA
+
+- fstStem:          kispêwât
+  generalWordClass: V
+  key:              kispêwâtêw
+  lemma:
+    sro: kispêwâtêw
+    cw:  kispêwâtêw
+  senses:
+    - category:   VTA-4
+      definition: s/he defends s.o., s/he is overprotective of s.o.
+  specificWordClass: VTA
+
+- fstStem:          ay-it
+  generalWordClass: V
+  key:              ay-itêw
+  lemma:
+    sro: ay-itêw
+    cw:  ay-itêw
+  senses:
+    - category:   VTA-5
+      definition: s/he says so to s.o, s/he says so of s.o.
+  specificWordClass: VTA
+
+- fstStem:          kâhtin
+  generalWordClass: V
+  key:              kâhtinam
+  lemma:
+    sro: kâhtinam
+    cw:  kâhtinam
+  senses:
+    - category:   VTI-1
+      definition: s/he obtains s.t., s/he attains s.t., s/he procures s.t.
+    - category:   VTI-1
+      definition: s/he pushes s.t.
+  specificWordClass: VTI
+
+- fstStem:          nahêkâcihtâ
+  generalWordClass: V
+  key:              nahêkâcihtâw
+  lemma:
+    sro: nahêkâcihtâw
+    cw:  nahêkâcihtâw
+  senses:
+    - category:   VTI-2
+      definition: s/he is sparing of s.t.
+  specificWordClass: VTI
+
+- fstStem:          oscoscocasi
+  generalWordClass: V
+  key:              oscoscocasiw
+  lemma:
+    sro: oscoscocasiw
+    cw:  oscoscocasiw
+  senses:
+    - category:   VTI-3
+      definition: s/he coughs a little
+    - category:   VTI-3
+      definition: s/he coughs up a little of something
+  specificWordClass: VTI
+
+- fstStem:          atimw
+  generalWordClass: N
+  key:              atim
+  lemma:
+    sro: atim
+    cw:  atim
+  senses:
+    - category:   NA-3
+      definition: dog
+    - category:   NA-3
+      definition: horse
+    - category:   NA-3
+      definition: beast of burden
+  forms:
+    - transcription:
+        sro: têm
+        cw:  têm
+    - transcription:
+        sro: atimw
+        cw:  atimw
+  specificWordClass: NA
+
+- fstStem:          atâhkw
+  generalWordClass: N
+  key:              atâhk
+  lemma:
+    sro: atâhk
+    cw:  atâhk
+  senses:
+    - category:   NA-3
+      definition: star
+  specificWordClass: NA
+
+- fstStem:          acâhkos
+  generalWordClass: N
+  key:              acâhkos
+  lemma:
+    sro: acâhkos
+    cw:  acâhkos
+  senses:
+    - category:   NA-1
+      definition: star, little star
+  specificWordClass: NA
+
+- fstStem:          nîmi
+  generalWordClass: V
+  key:              nîmiw
+  lemma:
+    sro: nîmiw
+    cw:  nîmiw
+  senses:
+    - category:   VAI-1
+      definition: s/he dances
+  specificWordClass: VAI
+
+- generalWordClass: N
+  key:              nîminâniwan
+  lemma:
+    sro: nîminâniwan
+    cw:  nîminâniwan
+  senses:
+    - category:   VAI-1 # INM?
```

Comment:
As far as I understand, this particular word-form would not be an INM (name-like particle).

For inflected word-forms for which we can identify an (FST-)lemmas from the FST analysis, I'm not entirely sure whether the entry should have a category at all, or whether the entry should inherit the category from that of the FST-lemma. Previously, I'd be inclined to opt for the former.

Anyhow, in our current itwêwina, we do not show the category for inflected forms, but only for the FST-lemmas, and that is a strategy that I've found appropriate.